README.md

```diff
@@ -40,6 +40,17 @@ checkerFramework {
 }
 ```
 
+The same example, using Kotlin syntax in a `build.gradle.kts` file:
+
+```kotlin
+configure<CheckerFrameworkExtension> {
```

Comment:
I know I didn't do this in #142, but we _might_ want this example to contain the fully-qualified name of `CheckerFrameworkExtension` (or add an import). In my case IntelliJ autocompleted it for me (and added the import automatically), but I don't know that we should assume that will work for everyone.

Reply:
I agree, and I've added an import statement here. Thanks for the suggestion!